### ISSUE TITLE
e/s 32 issue :

### ISSUE THREAD

**Issue by proof0zone** [sb53systems/G-Code-Flow-Temperature-Controller#7]

**Comment by sb53systems** [comment]
Send me your G-Code File at : sb53systems@gmail.com

**Comment by gueee** [comment]
Got the same error when hitting  the Generate button.
Do you want me to send the unaltered g-code file, since there is no output yet, or is there?

**Comment by sb53systems** [comment]
Sorry for the late reply, you can send me by email the Original G-Code, I'll let you know if I discover anything.  
  
Salim.

**Comment by sb53systems** [comment]
Is this problem still present?

**Comment by gueee** [comment]
> Is this problem still present?

Sorry, I couldn't find the time to retest it, but I'm going to today and will report.

**Comment by sb53systems** [comment]
Know that a new version was released today :  
https://github.com/sb53systems/G-Code-Flow-Temperature-Controller/releases/tag/V1.1Beta

**Comment by Nathan22211** [comment]
I just got the issue myself today

**Comment by sb53systems** [comment]
@Nathan22211 This problem occurs with the integration of Klipper_Estimator with this script, you have to reboot your computer or close manualy all CMD Consol in the Task Manager.  
![image](https://github.com/user-attachments/assets/d7887ffe-9b39-462e-89bb-cabdfd55683f)  
  
Let me know if you manage to resolve this issue.  
  
Best regards,  
Salim

**Comment by Nathan22211** [comment]
killing the console host nor rebooting fixed it.

**Comment by Nathan22211** [comment]
I do get an error about a float needing to be an int in french before this occurs though

**Comment by sb53systems** [comment]
> I do get an error about a float needing to be an int in french before this occurs though

I thought this error is fixed in the current version of the script!  
If you are on the latest version and this problem persists, send me the original G-Code and a screenshot of the error message.  
I'll do my best to resolve this essue.  
  
Best reguards,  
Salim

**Comment by Nathan22211** [comment]
![image](https://github.com/user-attachments/assets/7b59d4cc-3b92-4cb9-b9a3-cf68a9e644ce)

tested g code: https://gist.github.com/Nathan22211/fde16d1b3ca701afc340c66b8f25c0a2

**Comment by sb53systems** [comment]
@Nathan22211 Another script modified the G-Code before this one, the Script is not yet optimized to avoid comments added in the same line.  
![image](https://github.com/user-attachments/assets/f5e6c537-6b87-4fce-b3e8-b5f636b70408)  
  
Try one of these solutions temporarily:  
- Use this script first.
- Avoid comments in the same line in the other script.  
  
I will try to solve this problem soon in the future updates.
If you need more support, contact me in Discord: sb53systems  
  
Best reguards,  
Salim

**Comment by Nathan22211** [comment]
yeah just tried it straight out of orca slicer with it being the only post script and it still errors. I am a few versions behind and on a fork so maybe I should try to use mainline

**Comment by sb53systems** [comment]
> yeah just tried it straight out of orca slicer with it being the only post script and it still errors. I am a few versions behind and on a fork so maybe I should try to use mainline

@Nathan22211  Yeap, Forked version can cause essues, i recommend you use an official version.

**Comment by Nathan22211** [comment]
yeah just tried it with the latest mainline beta and still the same issue when running it from orca, and I'm pretty sure I turned verbose off so there shouldn't be as much of a comment issue. also it doesn't seem to find the extruder and used filament in the g code

**Comment by Nathan22211** [comment]
![image](https://github.com/user-attachments/assets/cacb7858-aa06-4cad-8fa5-41aaef8d923b)
[Bottom Panel_GPWC_R1_ABS_1h16m.txt](https://github.com/user-attachments/files/17249587/Bottom.Panel_GPWC_R1_ABS_1h16m.txt)

**Comment by sb53systems** [comment]
From where (which parameter) this comments come from !  
![image](https://github.com/user-attachments/assets/2f1f4683-35bd-4910-8a35-26a7653cb397)  
I'm using Orcaslicer v2.2.0-beta2!  
![image](https://github.com/user-attachments/assets/336d3898-2b9b-4429-92ef-de9725717a04)

**Comment by Nathan22211** [comment]
verbose gcode if I recall correctly. must have left it on by accident, though some other post scripts need it turned on so there's already a conflict there.

**Comment by Nathan22211** [comment]
turning verbose off fixes it but I'd definitely make it practically ignore anything after a ; in a line within the gcode blocks. some scripts, such as gradient infill, need it turned on to work properly

**Comment by sb53systems** [comment]
@Nathan22211 I will try to prepare an update as soon as possible, thank you for your contribution.  
Let me know if you found otherthing to improve.
